src/FSLibrary/MissionSorobanInvokeHostLoad.fs

```diff
@@ -33,10 +34,26 @@ let sorobanInvokeHostLoad (context: MissionContext) =
         (fun (formation: StellarFormation) ->
             formation.WaitUntilSynced [ coreSet ]
             formation.UpgradeProtocolToLatest [ coreSet ]
-            formation.UpgradeMaxTxSetSize [ coreSet ] 100000
+
+            let maxTxSetSize = 10000000
+            formation.UpgradeMaxTxSetSize [ coreSet ] maxTxSetSize
 
             formation.RunLoadgen coreSet context.GenerateAccountCreationLoad
-            formation.UpgradeSorobanLedgerLimitsWithMultiplier [ coreSet ] 1000
+            formation.UpgradeSorobanLedgerLimitsWithMultiplier [ coreSet ] 3000
             formation.UpgradeSorobanTxLimitsWithMultiplier [ coreSet ] 100
-            formation.RunLoadgen coreSet context.SetupSorobanInvoke
-            formation.RunLoadgen coreSet context.GenerateSorobanInvokeLoad)
+
+            // Have a unique instance per TX, assume 6 seconds per ledger for wiggle room
```

Comment:
mm, what does wiggle room mean here? you can set an arbitrarily high instance number, right?

Reply:
Yes, but it takes extra time to set up and extra memory in the load generating mode, so this value should be as low as possible. This "wiggle room" is just a conservative estimate such that each TX is in an independent instance, but it not so high as to havwe long setup times or high memory usage.